### ISSUE TITLE
References to autonumeric need to be updated

### ISSUE THREAD

**Issue by kalle-kilpi-midaxo** [autoNumeric/vue-autoNumeric#55]
Hello,

This component has great potential but using it is very inconvenient as it uses the outdated AutoNumeric case instead of the current lower case autonumeric naming convention. This forces complicated webpack configs as a workaround. Would it be possible to fix this issue?

Just doing a simple replace all "AutoNumeric" to "autonumeric" in the source JS file seems to fix the problem.

Best,
Kal

**Comment by kalle-kilpi-midaxo** [comment]
For those who need the alias workaround, put this in the vue.config.js file:

module.exports = {
  lintOnSave: false,
  configureWebpack: {
    resolve: {
      alias: {
        AutoNumeric: "autonumeric"
      }
    }
  }
};

**Comment by kalle-kilpi-midaxo** [comment]
For those who need the alias workaround, put this in the vue.config.js file:

```
module.exports = {
  lintOnSave: false,
  configureWebpack: {
    resolve: {
      alias: {
        AutoNumeric: "autonumeric"
      }
    }
  }
};

```

**Comment by arthurnassar** [comment]
@AlexandreBonneau can I pick up this issue please? I have a big project to maintain that has failing pipelines due to this issue

**Comment by xuhdev** [comment]
@arthurnassar You may not have noticed it but the author has thumbed up your comment :)

**Comment by arthurnassar** [comment]
@xuhdev yeah... I actually have something kinda ready but I'm having a difficult time on creating a branch to push it ... 
![image](https://github.com/autoNumeric/vue-autoNumeric/assets/98289378/f34fbf1b-92d3-492d-9db5-bf5f70cfbfa4)

Am I doing something wrong?

**Comment by AlexandreBonneau** [comment]
You should fork this repo, push your changes to your own fork, then from github, create a pull request from your repo to this one

**Comment by arthurnassar** [comment]
Aaah ok ok ... thanks, its my first contribution so I was not familiar with the flow

**Comment by arthurnassar** [comment]
Well... I tried a lot, and none if my attempts worked. Only changing references wouldn't be enough because of the backwards compatibility with the CDN right? anyone has any ideas that could help or at least give a light on this? Would really like to turn this library compatible with both

**Comment by AlexandreBonneau** [comment]
What do you mean by 'backwards compatibility with the CDN'?
What errors are you seeing, what did you try?

**Comment by arthurnassar** [comment]
Well by backwards compatibility I was meaning keep the lib working with both the CDN or the autonumeric on node_modules approach...
What I tried was to introduce a shim that would dinamically import the autonumeric either from the Window or from node_modules based on which of those were available...

Soon as I have some time I will post a detailed info of everything that I tried with the errors that I ran into.
